server/index.js

```diff
@@ -40,7 +40,7 @@ app.get("/api", (req, res) => {
 });
 
 app.get("*", (req, res) => {
-  res.sendFile(path.resolve(__dirname, "../client/build", "index.html"));
+  res.sendFile(path.resolve(__dirname, "../client/public", "index.html"));
```

Comment:
This was a mistake on my part that the main html file is in public not build. (copy paste error)